Apps/phcheckpointfirewall/checkpoint_connector.py

```diff
@@ -486,6 +487,38 @@ def _unblock_ip(self, param):
 
         return action_result.set_status(phantom.APP_SUCCESS, "Successfully unblocked {0}".format('subnet' if length != '32' else 'IP'))
 
+    def _delete_host(self, param):
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        if not(self._login(action_result)):
+            return action_result.get_status()
+
+        name = param.get('name', None)
+        uid = param.get('uid', None)
```

Comment:
```suggestion
        name = param.get('name')
        uid = param.get('uid')
```

`None` is the default value for `.get` if the key doesn't exist

Reply:
changes incorporated in 675481e